daffodil-propgen/src/main/resources/org/apache/daffodil/xsd/DFDL_part1_simpletypes.xsd

```diff
@@ -17,9 +17,9 @@
 -->
 
 <xsd:schema targetNamespace="http://www.ogf.org/dfdl/dfdl-1.0/"
-  xmlns:xsd="http://www.w3.org/2001/XMLSchema"
-  xmlns:dfdl="http://www.ogf.org/dfdl/dfdl-1.0/" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-  attributeFormDefault="unqualified" elementFormDefault="qualified">
```

Comment:
Do you mean we need to tolerate attributeFormDefault being duplicated here two times?  I deleted only the first duplicate so if we just need a single attributeFormDefault in some test to make sure it really is tolerated, we're still OK.

Reply:
Oops, attributeFormDefault indeed is different from elementFormDefault.  IntelliJ told me something about it being unused and I must've thought it was duplicated twice.  I'll check for any other test using it.  If none, I'll add a comment explaining why we use it here.

Reply:
I think if attributeFormDefault isn't provided it defaults to "unqualified". So this behavior should be the same. I guess it's just a matter of if we want to be explicit about it or not.

Reply:
Yes, it's just about whether DFDL processors complain and issue an SDE about having it. In principle they *could* do that because DFDL v1.0 doesn't use attributes at all. But... we found in practice that it's better to just ignore it being there.

Reply:
Daffodil already has a test to check that attributeFormDefault is tolerated in a DFDL schema (AttributeFormDefault.dfdl.xsd, SchemaDefinitionErrors.tdml, TestSDE.scala),  However, the file I edited isn't a DFDL schema - it's Daffodil's XML schema for DFDL.  So attributeFormDefault simply was being defined explicitly here, and I won't change that since several other files in this directory do the same thing too.